draft-ietf-quic-transport.md

```diff
@@ -2206,9 +2212,17 @@ The endpoint MUST use unpredictable data in every PATH_CHALLENGE frame so that
 it can associate the peer's response with the corresponding PATH_CHALLENGE.
 
 An endpoint MUST expand datagrams that contain a PATH_CHALLENGE frame to at
-least the smallest allowed maximum datagram size of 1200 bytes.  Sending UDP
-datagrams of this size ensures that the network path from the endpoint to the
-peer can be used for QUIC; see {{datagram-size}}.
+least the smallest allowed maximum datagram size of 1200 bytes, unless the
+anti-amplification limit for the path does not permit sending a datagram of
+this size.  Sending UDP datagrams of this size ensures that the network path
```

Comment:
It also seems like it's possible that we wouldn't be able to send a PATH_CHALLENGE at all, what should an endpoint do then?

Reply:
I'm not sure if there'd be such a case.

* Minimum size of a valid short header packet is 19 bytes (i.e. first-byte (1) + 0-byte CID (0) + 1-byte PN (1) + 1-byte payload (1) + AEAD tag (16)).
* Maximum size of a short header packet that contains PATH_CHALLENGE only is 50 bytes (i.e. first byte (1) + 20-byte CID (20) + 4-byte PN (4) + PATH_CHALLENGE (9) + AEAD tag (16)).

The latter (50) is less than the former multiplied by three (19*3=57).